app/src/main/java/org/schabi/newpipe/settings/BackupRestoreSettingsFragment.java

```diff
@@ -157,6 +202,36 @@ private void requestImportPathResult(final ActivityResult result) {
         }
     }
 
+    private void requestExportSubsResult(final ActivityResult result) {
+        if (result.getResultCode() == Activity.RESULT_OK && result.getData() != null) {
+            final Uri fileUri = result.getData().getData();
+            if (fileUri != null) {
+                final Intent intent = new Intent(
+                        requireContext(), SubscriptionsExportService.class);
+                intent.putExtra(SubscriptionsExportService.KEY_FILE_PATH, fileUri);
+                requireContext().startService(intent);
+                Toast.makeText(requireContext(), R.string.exporting, Toast.LENGTH_SHORT).show();
```

Comment:
Instead of showing a Toast I would do the same as in `SubscriptionFragment` and show an `ImportConfirmationDialog`. This will come naturally after deduplicating the code.